features/main/exception_to_status.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Using exception_to_status config
+  As an API developer
+  I can customize the status code returned if the application throw an exception
+
+  @createSchema
+  Scenario: Configure status code via the operation exceptionToStatus to map custom NotFound error to 404
+    When I add "Content-Type" header equal to "application/ld+json"
```

Comment:
Shouldn't the exception mapping also work for other content types?

Reply:
Yes it should, don't worry!
We cannot test every content types, having a functional test for this feature is already a very good thing.